icepyx/core/read.py

```diff
@@ -289,58 +291,122 @@ class Read:
 
     Examples
     --------
+    Reading a single file
+    ```
+    ipx.Read('/path/to/data/processed_ATL06_20190226005526_09100205_006_02.h5') # doctest: +SKIP
+    ```
+    Reading all files in a directory
+    ```
+    ipx.Read('/path/to/data/') # doctest: +SKIP
+    ```
+    Reading files that match a particular pattern (here, all .h5 files that start with `processed_ATL06_`).
+    ```
+    ipx.Read('/path/to/data/processed_ATL06_*.h5') # doctest: +SKIP
+    ```
+    Reading a specific list of files
+    ```
+    list_of_files = ['/path/to/data/processed_ATL06_20190226005526_09100205_006_02.h5', 
+                 '/path/to/more/data/processed_ATL06_20191202102922_10160505_006_01.h5']
+    ipx.Read(list_of_files) # doctest: +SKIP
+    ```
```

Comment:
```suggestion
    Reading a single file
    >>> ipx.Read('/path/to/data/processed_ATL06_20190226005526_09100205_006_02.h5') # doctest: +SKIP

    Reading all files in a directory
    >>> ipx.Read('/path/to/data/') # doctest: +SKIP

    Reading files that match a particular pattern (here, all .h5 files that start with `processed_ATL06_`).
    >>> ipx.Read('/path/to/data/processed_ATL06_*.h5') # doctest: +SKIP

    Reading a specific list of files
    >>> list_of_files = ['/path/to/data/processed_ATL06_20190226005526_09100205_006_02.h5', 
                 '/path/to/more/data/processed_ATL06_20191202102922_10160505_006_01.h5']
    >>> ipx.Read(list_of_files) # doctest: +SKIP
```

To match the docstring example formatting in other modules